server/controllers/availability.js

```diff
@@ -0,0 +1,101 @@
+const Availability = require("../models/availability");
```

Comment:
Some of this code seems to be from the prev PR, please rebase/adjust the base of this PR to reflect only the new changes

Reply:
Adjusted the PR base